<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualMachineViewDialog</class>
 <widget class="QDialog" name="VirtualMachineViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Register 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Integer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Register 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>String</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Floating Pt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="reg1Int">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="reg1Float">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="reg1String">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="reg2Int">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="reg2Float">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="reg2String">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Code</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="stepButton">
          <property name="text">
           <string>Step</string>
          </property>
          <property name="icon">
           <iconset resource="ideresources.qrc">
            <normaloff>:/icons/Images/StepOver2.png</normaloff>:/icons/Images/StepOver2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableWidget" name="codeGrid">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="rowCount">
             <number>5</number>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column>
             <property name="text">
              <string>#</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>OP</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Val</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Source</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="codeScroll">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Call stack</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTableWidget" name="callGrid">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Variables</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTableWidget" name="variableGrid">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Variable</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="variableScroll">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Heap</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="heapGrid">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Int</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Float</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>String</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="heapScroll">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Stack</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableWidget" name="stackGrid">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Int</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Float</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>String</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="stackScroll">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>reg1Int</tabstop>
  <tabstop>reg1Float</tabstop>
  <tabstop>reg1String</tabstop>
  <tabstop>reg2Int</tabstop>
  <tabstop>reg2Float</tabstop>
  <tabstop>reg2String</tabstop>
  <tabstop>stepButton</tabstop>
  <tabstop>codeGrid</tabstop>
  <tabstop>callGrid</tabstop>
  <tabstop>variableGrid</tabstop>
  <tabstop>heapGrid</tabstop>
  <tabstop>stackGrid</tabstop>
 </tabstops>
 <resources>
  <include location="ideresources.qrc"/>
 </resources>
 <connections/>
</ui>
